perl-package/AI-MXNetCAPI/mxnet.i

```diff
@@ -338,6 +351,37 @@ int MXNDArrayCreateEx(const mx_uint *in,
                               int delay_alloc,
                               int dtype,
                               NDArrayHandle *out);
+/*!
```

Comment:
Does this file have to be updated when c_api is changed each time? Is this not automated?

Reply:
It may look like it could be automated but it can not.
It's not just a copy/paste from the c_api.h,  it's  a SWIG interface file and the variables names are changed to denote a way of how they should be treated to make c api to communicate with perl.
Another file called mxnet_typemaps.i in the dir has actual glue code that depends on the vars.
It's just another version of 'ctypes' .